cdap-hbase-compat-0.96/src/main/java/co/cask/cdap/data2/util/hbase/HBase96TableUtil.java

```diff
@@ -41,6 +40,7 @@
 /**
  *
  */
+@SuppressWarnings("ConstantConditions")
```

Comment:
I will need to move it to every method. Hence I moved it to the top of the class. let me know if that is not a good idea.